common/helpers/front_end.py

```diff
@@ -16,3 +17,8 @@ def get_page_section(url):
     if match is not None:
         groups = match.groups()
         return groups[0]
+
+
+def get_clean_url(url):
```

Comment:
As a hotfix, this works (I tested it replacing `&amp;` and `&equals;` and it did both) though it fails on something like `&#61;` or `&#63` entity numbers.

I know in js unescape() has been [functionally deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape) in favor of decodeURI, so I'm wondering if - for long-term use of this function  - we should consider `urllib.parse` or similar rather than just an unescape?

Reply:
We can look into this some more, but I haven't seen any instances where entity numbers get into the url so that can be a future action item.